README.md

```diff
@@ -10,8 +10,8 @@ To clone this repo and get started, run:
 ```
 git clone https://github.com/deephaven-examples/deephaven-matplotlib-base.git
 cd deephaven-matplotlib-base
-docker-compose pull
-docker-compose up  --build -d
+docker compose pull
+docker compose up --build -d
```

Comment:
There's a line at the bottom of the README that we should remove since we removed the examples:

> The 3D examples in this readme are from [Rashida Nasrin Sucky](https://towardsdatascience.com/five-advanced-plots-in-python-matplotlib-134bfdaeeb86) the data is available from [kaggle](https://www.kaggle.com/fazilbtopal/auto85)